pkg/apis/externaldns/types_test.go

```diff
@@ -294,7 +294,7 @@ func TestParseFlags(t *testing.T) {
 			title: "override everything via environment variables",
 			args:  []string{},
 			envVars: map[string]string{
-				"EXTERNAL_DNS_MASTER":                          "http://127.0.0.1:8080",
+				"EXTERNAL_DNS_APISERVER":                          "http://127.0.0.1:8080",
```

Comment:
This needs a gofmt.

Reply:
I really don't think so, I did it, even in a dedicated commit: https://github.com/kubernetes-sigs/external-dns/pull/1636/commits/9e194276c8c8ad5ed9cdda3732208b9e73387b6f